### PR TITLE
Add basic MIDI latency testing

### PR DESCRIPTION
**Pull request by BenjaminFair**
As requested in issue #3.
Support for taking multiple samples and computing statistics will be added later.

### CODE REVIEW

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    public void onPause() {
+        logger.broadcastManager.unregisterReceiver(mLogReceiver);
+        super.onPause();
+    }
+
+    @Override
+    public void onClick(View v) {
+        if(mMidiDevice == null) {
+            logger.log("Wait until the MIDI devices connects");
+        } else {
```
Comment (kamrik) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: Returning in the "if" and thus avoiding the need for "else" makes for a bit more readable code (and less unnecessary indentation).

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java [hunk 2]
```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
```
Comment (kamrik) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: Year in copyright header

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+    private BroadcastReceiver mLogReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            String msg = intent.getStringExtra("message");
+            mTextView.append(msg + "\n");
+        }
+    };
+
+    private void setupMidiOut() {
+        mInputPort = mMidiDevice.openInputPort(0);
```
Comment (kamrik) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: We need some comment somewhere about the "in" vs "out" confusion. Why *MidiOut functions use mInputPort and vice versa.

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java (hunk 1 above)
Comment (BenjaminFair) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: Done

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java (hunk 2 above)
Comment (BenjaminFair) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: Done

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java (hunk 3 above)
Comment (BenjaminFair) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiFragment.java]: Done

**Comment by gkasten** [comment]
units on symbols especially times / delays
"if("

**Pull request merged**